src/lib.rs

```diff
@@ -425,6 +426,48 @@ impl Account {
         })
     }
 
+    /// Load a new account by private key, with a default or custom HTTP client
+    ///
+    /// https://www.rfc-editor.org/rfc/rfc8555#section-7.3.1
+    ///
+    /// the `DefaultClient::try_new()?` can be used as the http client by default.
+    /// The returned [`AccountCredentials`] can be serialized and stored for later use.
+    /// Use [`Account::from_credentials()`] to restore the account from the credentials.
+    #[cfg(feature = "hyper-rustls")]
```

Comment:
Still don't really want to have both the `_http()` and non-`_http()` variants. Which one of these do you want to use?